samples/client/petstore/java/rest-assured/src/test/java/io/swagger/client/api/PetApiTest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Swagger Petstore
+ * This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\
+ *
+ * OpenAPI spec version: 1.0.0
+ * Contact: apiteam@swagger.io
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+
+package io.swagger.client.api;
+
+import java.io.File;
+import io.swagger.client.model.ModelApiResponse;
+import io.swagger.client.model.Pet;
+import io.swagger.client.ApiClient;
+import io.swagger.client.api.PetApi;
+import io.restassured.builder.RequestSpecBuilder;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Ignore;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+import static io.restassured.http.ContentType.JSON;
+import static io.restassured.mapper.ObjectMapperType.GSON;
+
+/**
+ * API tests for PetApi
+ */
+@Ignore
+public class PetApiTest {
+
+    private PetApi api;
+
+    @Before
+    public void createApi() {
+        api = ApiClient.api(ApiClient.Config.apiConfig().reqSpecSupplier(
+                () -> new RequestSpecBuilder()
+                        .setContentType(JSON)
+                        .setBaseUri("http://petstore.swagger.io:80/v2"))).pet();
+    }
+
+    /**
+     * Invalid input
+     */
+    @Test
+    public void shouldSee405AfterAddPet() {
+        Pet body = null;
```

Comment:
The petstore samples should include actual tests against the petstore server to verify the generated code in CI

Reply:
Thanks a lot. I did it in https://github.com/swagger-api/swagger-codegen/pull/7492/commits/c698765bdac234fd839f9626b5a96bc402ff1f25